example/lib/main.dart

```diff
@@ -85,6 +91,7 @@ class MyApp extends StatelessWidget {
           initialRoute: '/',
           routes: {'/': (_) => const MyHomePage()},
           theme: ThemeData(
+            micaBackgroundColor: Colors.transparent,
```

Comment:
This shouldn't be handled this way. We should check what's the current `AppTheme.acrylicEffect` to get the best color for each.

When this is transparent, we get a weird effect when it's `WindowEffect.disabled`

Reply:
Do I understand correctly that then it is better to use this option?
`micaBackgroundColor: appTheme.acrylicEffect == WindowEffect.acrylic || appTheme.acrylicEffect == WindowEffect.mica ? Colors.transparent : null,`